include/nuttx/mtd/mtd.h

```diff
@@ -45,11 +45,6 @@
                                            *      receive geometry data (see mtd.h)
                                            * OUT: Geometry structure is populated
                                            *      with data for the MTD */
-#define MTDIOC_XIPBASE    _MTDIOC(0x0002) /* IN:  Pointer to pointer to void in
```

Comment:
I believe it is worth adding a note here, similar to this one in Linux:
https://elixir.bootlin.com/linux/latest/source/include/uapi/mtd/mtd-abi.h#L152

Reply:
Done.